### PR TITLE
Update package_repo.rb

### PR DESCRIPTION
**Pull request by whysthatso**
This combination of key and keyserver url works much more predictable

### CODE REVIEW

**Comment by whysthatso** [comment]
related to https://github.com/phlipper/chef-percona/issues/339

File: attributes/package_repo.rb [hunk 1]
```diff
@@ -11,8 +11,8 @@
   "default" => node["platform_version"].to_i
 )
 
-default["percona"]["apt"]["key"] = "0x1C4CBDCDCD2EFD2A"
-default["percona"]["apt"]["keyserver"] = "hkp://keys.gnupg.net:80"
+default["percona"]["apt"]["key"] = "1C4CBDCDCD2EFD2A"
+default["percona"]["apt"]["keyserver"] = "hkp://keys.gnupg.net"
```
Comment (jharley) [inline attributes/package_repo.rb]: The ':80' was added here to deal with nodes with more restricted outbound traffic policy.  `hkp` uses port 11371 by default

**Comment by canadianveggie** [comment]
Please strip off the 0x. With it the fingerprint never matches so apt cookbook always tries to re-install the key. With the flakiness of of the keyserver (https://github.com/phlipper/chef-percona/issues/235), this causes a lot of chef failures in our system.

**Comment by whysthatso** [comment]
@jharley would my additions address your concern?

**Comment by jmara** [comment]
@whysthatso can you replace the single with double quotes? Then this change should pass the Travis CI build. And hopefully someone merges it to master ;)

**Comment by whysthatso** [comment]
hth :)

**Comment by damacus** [comment]
Any chance you can take another look at this PR if it's still failing for you?

**Comment by damacus** [comment]
Closing due to inactivity. 

If this is still an issue please reopen or open another issue. Alternatively drop by the #sous-chefs channel on the [Chef Community Slack](http://community-slack.chef.io/) and we'll be happy to help! 

Thanks,
Sous-Chefs

**Pull request closed without merging**